.config/ags/modules/.configuration/user_options.js

```diff
@@ -71,11 +71,46 @@ let configOptions = {
         'dateFormat': "%d/%m", // On notif time
     },
     'weather': {
-        'city': "",
+        'city': "Chengdu",
     },
     'workspaces': {
         'shown': 10,
     },
+    'dock': {
+        'enabled': false,
+        // Threshold for hover to trigger dock display
+        'hoverMinHeight': 5,
+        'pinnedApps': ['firefox', 'org.gnome.Nautilus'],
+        // top or bottom
+        'layer': 'top',
+        // Find the window's icon by its class with levenshteinDistance
+        // All file paths are preprocessed and stored at ags startup, so if there
+        // are so many files under the path it will affect performance
+        // Maybe you need a comprehensive icon theme
+        // Example: ['/usr/share/icons/Tela-nord-dark/scalable/apps', 'others...']
+        'iconSearchPaths': [''],
```

Comment:
let's move this to an 'icon' category since i wanna have this icon guessing for overview too